lib/db.js

```diff
@@ -1,6 +1,10 @@
-const initialArticles = [
+const articles = [
   {
     id: "24343",
+    thema: "politics",
```

Comment:
you could write "topic" instead of "thema" to stay english :)

Reply:
yep hehe